TLV/src/tlv.c

```diff
@@ -23,6 +21,18 @@ void tlv_initI2sPins()
 	GPIO_PinAFConfig(GPIOA, SD_I2S_SD, SD_I2S_ALT_FUNC);
 }
 
+void I2S_write(RingBuffer* I2S_buffer)
+{
+	while(!SPI_I2S_GetFlagStatus(SPI2, SPI_I2S_FLAG_TXE));
+	SPI_I2S_SendData(SPI2, get(I2S_buffer));
+}
+
+void I2S_read(SPI_TypeDef* SPIx)
+{
```

Comment:
Now that I think about it, perhaps this and the I2S_write function could have the naming convention with the prefix SD_. Any number of people could use the prefix I2S.